backend/accounts.sql

```diff
@@ -0,0 +1,13 @@
+-- @block 
+CREATE TABLE IF NOT EXISTS account (
```

Comment:
How is this run? Is it ran on start up?

Reply:
Thsi is simply the mysql command, the actual database comes later